src/main/java/thePackmaster/patches/lockonpack/AdditiveLockOnPatch.java

```diff
@@ -0,0 +1,60 @@
+package thePackmaster.patches.lockonpack;
+
+import com.evacipated.cardcrawl.modthespire.lib.SpirePatch2;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.orbs.AbstractOrb;
+import com.megacrit.cardcrawl.powers.LockOnPower;
+import javassist.CannotCompileException;
+import javassist.expr.ExprEditor;
+import javassist.expr.FieldAccess;
+import thePackmaster.powers.lockonpack.TunnelVisionPower;
+import thePackmaster.util.Wiz;
+
+public class AdditiveLockOnPatch {
+
+    public static int applyMultiLockOn(AbstractMonster m, int damage)
+    {
+        float damageModifier = 1.5F
+                + 0.25F
+                * (Wiz.getLogicalPowerAmount(m, LockOnPower.POWER_ID) - 1);
```

Comment:
Does this work correctly if you ahve no lock-on currently? This probably won't be called then but just making sure you thought about that.

Doesn't this need a check for Tunnelvision being active?

Reply:
It's only getting called if lock-on is on... but it does need tunnelvision check.